### PR TITLE
New pictures and 1st rev of us4R to us4R-lite migration [WIP]

### PR DESCRIPTION
**Pull request by heymarcin**
This is the very first version of the migrated us4R to us4R-lite user manual -- changed pictures, modified description specific to us4R.

**Still TODO:**

- Make pictures:
   - us4r-lite-power.jpg -- adapter + system
   - us4r-lite-probe-conn.jpg -- top view with probe adapter
   - us4r-lite-pcie-back+cables -- back with PCIe cables connected
   - us4r-lite-pcie-back-io -- back with marked I/Os

- Verify `tech-spec.md`:
   - Power consumption (average) 70W
   - ...

### CODE REVIEW

**Comment by heymarcin** [comment]
@beatawitek this PR is updated - Please review!

File: docs/content/first-use.md [hunk 1]
```diff
@@ -1,40 +1,138 @@
...
+* an external host PC have been connected to the **us4R-lite™**.
 
 Step-by-step instruction:
 
-1.  Connect the mains to the **us4R-lite™** and turn the *Power Switch* on.
-
+1.  Connect the supplied power adapter to the **us4R-lite™** (DC jack) and to the mains (the power cable).
 2.  Now, turn on the device by clicking the ON/OFF button.
-
 3.  Connect the ultrasound probe.
```
Comment (pjarosik) [inline docs/content/first-use.md]: @heymarcin Perhaps swap points 2. and 3.?

File: docs/content/first-use.md [hunk 2]
```diff
@@ -1,40 +1,138 @@
...
+If you cannot find the adapter that suits your application, it is possible to order a custom probe adapter from the us4us®. Please contact us at <support@us4us.eu> to discuss the options.
+
+
+(set-up/System setup)=
+## System setup
+
+The **us4R-lite™** should be positioned so that operation is safe --- i.e. on a stable, flat surface in a place with no risk of spillage on the device and away from the sources of interference and radiation. 
+To limit unintentional radiation from the device, the provided ferrites for PCIe cables should be mounted on each cable, on both ends of the cable and as close to the connectors as possible. The external power supply and/or power strip should be placed nearby the device. 
+
+<!-- For more details see section: {numref}`Section %s <hardware/general>`. -->
```
Comment (pjarosik) [inline docs/content/first-use.md:88]: Preferred syntax for inserting comments: https://myst-parser.readthedocs.io/en/latest/syntax/typography.html#comments

**Review by pjarosik — COMMENTED** [review]
Reviewed first-use.md only

**Review by pjarosik — APPROVED** [review]

**Pull request merged**